functions/prepare-check/function.json

```diff
@@ -9,7 +9,7 @@
       "name": "queueMessage",
       "type": "queueTrigger",
       "direction": "in",
-      "queueName": "prepare-check",
+      "queueName": "jon-prepare-check",
```

Comment:
this can be standardised back to `prepare-check` now we have everything in place